test/testvfdswmr.sh.in

```diff
@@ -571,24 +571,21 @@ done
 # read and written by VFD SWMR.
 #
 if [ ${do_zoo:-no} = yes ]; then
-	[ -e ./fifo ] && rm -f ./fifo
-	mkfifo -m 0600 ./fifo
 	rm -f ./shared_tick_num
 	echo launch vfd_swmr_zoo_writer
-	STDIN_PATH="./fifo" catch_out_err_and_rc vfd_swmr_zoo_writer \
-	    ../vfd_swmr_zoo_writer -m 1000 -q &
+	catch_out_err_and_rc vfd_swmr_zoo_writer \
+	    ../vfd_swmr_zoo_writer -l 5 -m 1000 -q &
```

Comment:
`-l 5` tells the writer that it has 5 seconds to finish the creation or deletion of objects. `-l 10` tells the reader that it has 10 seconds to validate the creation or deletion of objects.

Reply:
`-l` is the maximal interval (between the writer's finishing zoo creation or deletion and reader's finishing validation) in terms of tick now.